tasks/main.yml

```diff
@@ -27,4 +27,4 @@
     dest: /etc/apt/preferences.d/deb_nodesource_com_node.pref
 
 - name: Install Node.js
-  apt: pkg=nodejs={{ nodejs_version }}.* state=installed update_cache=yes
+  apt: pkg=nodejs state=installed update_cache=yes
```

Comment:
i may have misunderstood the purpose of this line. i edited it for node 4.x, verified it works with 4.x and 0.10.x